tests/issues/issue-1652.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+return [
+    'issue-1652' => [
+        'ua' => 'Mozilla/5.0 (PlayStation Vita 3.65) AppleWebKit/537.73 (KHTML, like Gecko) Silk/3.2',
+        'properties' => [
+            'Comment' => 'PlayStation Vita',
+            'Browser' => 'Sony Playstation Vita',
```

Comment:
Could we change this to be just "Playstation Browser" like the other ones in the playstation.json file?

I'm not sure why this one was separated out. I think it's easy enough to look at the platform/device and know that it's for the Vita.